lte/gateway/python/integ_tests/s1aptests/s1ap_utils.py

```diff
@@ -1117,11 +1137,48 @@ def disable_service(self, service):
         Args:
             service: (str) service to disable
         """
+        service_name = self.get_service_name_from_init_system(service)
         if self._init_system == InitMode.SYSTEMD:
-            self.exec_command(f"sudo systemctl mask magma@{service}")
-            self.exec_command(f"sudo systemctl stop magma@{service}")
+            self.exec_command(f"sudo systemctl mask {service_name}")
+            self.exec_command(f"sudo systemctl stop {service_name}")
         elif self._init_system == InitMode.DOCKER:
-            self.exec_command(f"docker stop {service}")
+            # TODO GH14055
+            # Same argument as above: The container interdependencies
+            # are handled manually at the moment
+            #
+
+            if (
+                service_name == "oai_mme"
+                or service_name == "sessiond"
+                or service_name == "mobilityd"
+                or service_name == "pipelined"
+            ):
+                self.exec_command(
+                    "docker restart oai_mme mobilityd sessiond "
+                    "connectiond pipelined envoy_controller",
```

Comment:
Isn't this function supposed to disable the services? I think a simple `docker stop {service}` should be enough, as in the `else` bracket. Restarting is not what we want here.

Reply:
Yes, thank you. Copy-paste bug :expressionless: